steps/27/README.md

```diff
@@ -82,34 +85,33 @@ QUnit.test("Should return the translated texts", (assert) => {
 ### webapp/test/unit/unitTests.qunit.ts \(New\)
 
 We create a new `unitTests.qunit.ts` file under `webapp/test/unit/`.
-This script loads and executes our formatter test. Before the QUnit test execution can be started we need to wait until the Core has been booted. Therefore, you need to disable the autostart `QUnit.config.autostart = false;`, require the `sap/ui/core/Core` module and use `Core.ready()` to wait until the Core has booted and then you can start the QUnit tests with `QUnit.start()`.
+This module is the entrypoint for all our unit tests and will be referenced in the test suite that will be created later on.
```

Comment:
Somehow the explanation about the implementation in the file itself got lost. How about to change the description to:

This module will serve as the entry point for all our unit tests. It will be referenced in the test suite that we will set up later on.

Inside the  `unitTests.qunit.ts` file, we import the unit test for the custom formatter. This ensures that any tests related to the custom formatter functionality will be included when running our unit tests.

Reply:
Thanks 👍🏻 I've update it as suggested.